frontend/packages/ceph-storage-plugin/src/components/independent-dashboard-page/status-card/card.tsx

```diff
@@ -0,0 +1,86 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { GalleryItem, Gallery } from '@patternfly/react-core';
+import { ALERTS_KEY } from '@console/internal/actions/dashboards';
+import { withDashboardResources } from '@console/internal/components/dashboard/with-dashboard-resources';
+import { PrometheusRulesResponse, alertURL } from '@console/internal/components/monitoring';
+import { getAlerts } from '@console/shared/src/components/dashboard/status-card/alert-utils';
+import { filterCephAlerts } from '@console/ceph-storage-plugin/src/selectors';
+import AlertItem from '@console/shared/src/components/dashboard/status-card/AlertItem';
+import AlertsBody from '@console/shared/src/components/dashboard/status-card/AlertsBody';
+import { PrometheusResponse } from '@console/internal/components/graphs';
+import DashboardCard from '@console/shared/src/components/dashboard/dashboard-card/DashboardCard';
+import DashboardCardTitle from '@console/shared/src/components/dashboard/dashboard-card/DashboardCardTitle';
+import HealthBody from '@console/shared/src/components/dashboard/status-card/HealthBody';
+import DashboardCardHeader from '@console/shared/src/components/dashboard/dashboard-card/DashboardCardHeader';
+import DashboardCardBody from '@console/shared/src/components/dashboard/dashboard-card/DashboardCardBody';
+import HealthItem from '@console/shared/src/components/dashboard/status-card/HealthItem';
+import { StorageDashboardQuery, STORAGE_HEALTH_QUERIES } from '../../../constants/queries';
+import { getCephHealthState } from '../../dashboard-page/storage-dashboard/status-card/utils';
+
+const cephStatusQuery = STORAGE_HEALTH_QUERIES[StorageDashboardQuery.CEPH_STATUS_QUERY];
+
+const Alerts = withDashboardResources((props) => {
```

Comment:
this component is the same as `CephAlerts` https://github.com/openshift/console/blob/master/frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/status-card/status-card.tsx#L30 . Why not just export that one and reuse it here ?